sbi/utils/user_input_checks_utils.py

```diff
@@ -9,6 +9,19 @@
 from torch.distributions import Distribution, constraints
 
 
+def get_distribution_parameters(dist, device):
+    '''Used to get the tensors of the parameters in torch distributions.
+    Returns the tensors relocated to device.
```

Comment:
I find this comment confusing, as it does not relocate anything. How about `This function is used to get tensors which have to be relocated onto a different device when the .to() method is called`.